src/unix/bsd-proctitle.c

```diff
@@ -28,7 +28,7 @@
 
 static uv_mutex_t process_title_mutex;
 static uv_once_t process_title_mutex_once = UV_ONCE_INIT;
-static char* process_title;
+static char* process_title = NULL;
```

Comment:
These are unnecessary since they're static.

Reply:
True, although `aix.c` has explicit `NULL` initialization on the relevant static variables. Is leaving out the `= NULL` preferred?

https://github.com/libuv/libuv/blob/36549815eed297c74448ff580b5c84713245ec5d/src/unix/aix.c#L70-L73

Reply:
Yes. aix.c is non-idiomatic libuv in that respect (and others.)